Reneiski_Homework1/ReneiskiSelenium11/Tests/Tests.cs

```diff
@@ -0,0 +1,121 @@
+﻿using NUnit.Framework;
+using ReneiskiSelenium11.PageObjects.ButtonPage;
+using ReneiskiSelenium11.PageObjects.CheckBoxPage;
+using ReneiskiSelenium11.PageObjects.LinksPage;
+using ReneiskiSelenium11.PageObjects.RadioButtonPage;
+using ReneiskiSelenium11.PageObjects.WebTablePage;
+using ReneiskiSelenium11.PageObjects.WebTablePage.RegistrationForm;
+
+namespace ReneiskiSelenium11.Tests
+{
+    public class Selenium : BaseTest
```

Comment:
I think better to name test class according to the tests you have in this class

Reply:
Fixed